CHANGELOG.md

```diff
@@ -3,8 +3,11 @@ All notable changes to this add-on will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [Unreleased]
+CAUTION - While we do review all scripts to ensure they don't do anything obviously malicious, you should still review them and use them with caution.
```

Comment:
Will this show up anywhere in the ZAP UI?

Reply:
No.

Reply:
Maybe this one should be moved to the help page.

Reply:
Okay,will do that